sirepo/template/omega.py

```diff
@@ -283,11 +253,49 @@ def _generate_parameters_file(data):
     return res + template_common.render_jinja(SIM_TYPE, v)
 
 
+def _output_info(run_dir):
+    def _report_info(sim_count, model_name, report_count):
+        return PKDict(
+            simCount=sim_count,
+            modelName=model_name,
+            reportCount=report_count,
+            modelKey=f"{model_name}{sim_count}-{report_count}",
+        )
+
+    res = []
+    idx = 0
+    while True:
```

Comment:
maybe something like this
```python
sim_dir = run_dir.join(f"run{idx + 1}")
while sim_dir.exists() and _has_file():
    ...
    idx += 1
    sim_dir = run_dir.join(f"run{idx + 1}")
```
instead of the `break`

Reply:
Yes, that's a good idea. The repetition of f"run{idx + 1}" around the module isn't good either, so I'll replace that with a method as well.